pkg/pillar/vendor/github.com/packetcap/go-pcap/pcap_linux.go

```diff
@@ -243,7 +255,9 @@ func (h *Handle) Close() {
 	logger := log.WithFields(log.Fields{
 		"iface": h.iface,
 	})
-	h.isOpen = false
+	// this is the only call to set isOpen that happens when a different
+	// goroutine might be running, so we wrap it
+	atomic.StoreUint32(&h.isOpen, 1)
```

Comment:
Are you deferring the wait for the reader to a separate PR?

Reply:
Yes. Let's discuss in an issue on pcap